eng/Versions.props

```diff
@@ -15,7 +15,8 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
   <PropertyGroup>
-    <VersionPrefix>16.3.0</VersionPrefix>
+    <MSBuildVersionMajorMinor>16.3</MSBuildVersionMajorMinor>
+    <VersionPrefix>$(MSBuildVersionMajorMinor).0</VersionPrefix>
```

Comment:
Can you revert this block? It's no longer needed because we can hack less below.